calicolipidlibrary/MIP2C.py

```diff
@@ -45,39 +45,23 @@ def theoreticalDigest(self):
 			return(FRAGMENTS)
 
 
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
+
+		def generateLibrary(self, target=None, mode="pos"):
 			if target: handle = open(target, 'a+')
-			if mode == "pos":  
+			if mode == "pos":
 				adduct_set = self.pos_adduct_set
 			elif mode =="neg":
 				adduct_set = self.neg_adduct_set
 			for c1 in self.chain1_ranges:
 				for c2 in self.chain2_ranges:
 					for adduct in adduct_set:
-						x  = MIP2C("MIP2C",[c1[0],c2[0]], [c1[1],c2[1]], [c1[2],c2[2]],  adduct=adduct)						
-						content = x.printNist()
-						if target: handle.write(content)
-						else: sys.stdout.write(content)
-			if target: handle.close()
-
-
-
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
```

Comment:
This is the second definition of generateLibrary() which shadows the first, hiding its customized chains behavior (which has a syntax error)